slsa/buildtype.go

```diff
@@ -84,9 +84,9 @@ func (b *GithubActionsBuild) BuildConfig(context.Context) (interface{}, error) {
 }
 
 func addEnvKeyString(m map[string]interface{}, k string, v string) {
-	if v != "" {
-		m[k] = v
-	}
+	// Always record the value, even if it's empty. Let
```

Comment:
If empty values should be allowed, should `i.Environment` always be set?

https://github.com/slsa-framework/slsa-github-generator/blob/b9e63b71c0f8f5a1de31d024a7ca5e9ef7ea3f0f/slsa/buildtype.go#L207-L209

Reply:
Will do. In practice this code would probably never be hit, since there will always have some non-empty values.

Reply:
Done.